app/gui/src/model/project/synchronized.rs

```diff
@@ -488,6 +488,14 @@ impl Project {
                         content_roots.remove(id);
                     }
                 }
+                Event::Notification(Notification::VisualisationEvaluationFailed(update)) => {
+                    error!(
+                        logger,
+                        "Visualisation evaluation failed in context {update.context_id} \
+                        for visualisation {update.visualisation_id} of expression \
+                        {update.expression_id}. Error: {update.message}."
```

Comment:
Likely there should be no trailing dot, as we expect error message to be a full sentence.

Reply:
oh, cool, thanks! done 🙂